examples/app_rgb_pwm_lpc17xx.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2015, Paul Osborne <osbpau@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
```

Comment:
Any chance this can be single-licensed under Apache so that we have similar terms for all the demo code?